src/components/Counter/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @copyright Copyright (c) 2019 Marco Ambrosini <ma12co@pm.me>
+ *
+ * @author Marco Ambrosini <ma12co@pm.me>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import ScopeComponent from 'Utils/ScopeComponent'
+import Counter from './Counter'
+
+ScopeComponent(Counter)
```

Comment:
This should not be needed. 
We only use it for external libraries we do not have control of.
```suggestion
```